app/web/src/newhotness/logic_composables/management.ts

```diff
@@ -0,0 +1,44 @@
+import { computed, ComputedRef, inject, ref } from "vue";
+import { useQuery } from "@tanstack/vue-query";
+import { FuncKind, FuncRun } from "../api_composables/func_run";
+import { ManagementFuncJobState } from "../api_composables/management_func_job_state";
+import { useApi, routes } from "../api_composables";
+import { assertIsDefined, Context } from "../types";
+
+export const useManagementFuncJobState = (
```

Comment:
Need to double check that this can be re-used by multiple cards and can have shared state when dealing with the same func run